src/google/protobuf/compiler/cpp/enum.cc

```diff
@@ -302,7 +302,7 @@ void EnumGenerator::GenerateSymbolImports(io::Printer* p) const {
         {
             Sub("VALUE", EnumValueName(enum_->value(j))).AnnotatedAs(value),
             {"DEPRECATED",
-             value->options().deprecated() ? "PROTOBUF_DEPRECATED_ENUM" : ""},
+             value->options().deprecated() ? "PROTOBUF_DEPRECATED" : ""},
```

Comment:
I believe that I need to run `generate_descriptor_proto.sh` to regenerate prebuilt `pb{.cc,.h}` files, but running the script gives the changes in php sources only.

Reply:
I got it.
There are no deprecated enum values in builtin proto files, hence there is no need to regenerate them.